Makefile

```diff
@@ -33,9 +32,9 @@ change-context:
 check-secrets:
 	@$(eval ACTUAL=$(shell [ -z $(shell readlink Secrets) ] && echo "<empty>" || echo $(shell readlink Secrets)))
 	@[ "$(ACTUAL)" == "Secrets-$(DEPLOY)" ] || \
-	(printf "\033[1mUnexpected Secrets link\033[0m\n"; \
-	 printf "\033[92mExpected Secrets:\033[0m Secrets-$(DEPLOY)\n"; \
-	 printf "\033[93mActual Secrets:\033[0m   $(ACTUAL)\n"; \
```

Comment:
Should we keep this formatting? I like the bold and the colors :)

Reply:
10-4 — reverting to keep formatting...